views/js/controller/DeliveryServer/awaiting.js

```diff
@@ -47,6 +47,44 @@ define([
      */
     var cssScope = '.awaiting-authorization';
 
+    /**
+     * Nothing needs to be taken to system clipboard on proceed action
+     */
+    var cleanClipboard = function cleanClipboard () {
```

Comment:
This function sounds like a separate feature for me, and not something specific for the `DeliveryServer/awaiting.js`. It would make sense to create a global helper for that. What do you think?

Reply:
Not sure that we need this, maybe different library to control clipboards, but this won't be such a fast solution. (I guess that it will be a task for 8 story points with architect review, and for the core team).